content/courses/program-optimization/lookup-tables.md

```diff
@@ -5,65 +5,81 @@ objectives:
   - Create lookup tables
   - Extend lookup tables
   - Use lookup tables with versioned transactions
-description: "Use large amounts of accounts by using lookup tables."
+description:
+  "Harness a great deal of accounts through the application of lookup tables."
+keywords:
+  - lookup tables
+  - address
+  - transaction versions
+  - versioned
+  - legacy
 ---
 
 ## Summary
 
-- **Versioned Transactions** refers to a way to support both legacy versions and
-  newer versions of transaction formats. The original transaction format is
-  "legacy" and new transaction versions start at version 0. Versioned
-  transactions were implemented to support the use of Address Lookup Tables
-  (also called lookup tables or LUTs).
-- **Address Lookup Tables** are accounts used to store addresses of other
-  accounts, which can then be referenced in versioned transactions using a 1
-  byte index instead of the full 32 bytes per address. This enables the creation
-  of more complex transactions than what was possible prior to the introduction
-  of LUTs.
+- **Versioned Transactions** are like different versions of a recipe that allow
+  us to use both old and new ingredients. In this case, the "recipes" are
+  transaction formats. The old version, known as "legacy," and the new versions
+  start from version 0. We made these new versions to work better with a new
+  tool called Address Lookup Tables (LUTs).
```

Comment:
> **Versioned Transactions** are like different versions of a recipe that allow
  us to use both old and new ingredients. 
  
 Is too complex. Wedon't need a cooking analogy. People understand versions. The only thing we need to convey is that if they want to use lots of accounts, they need a V0 transaction not a legacy transaction.